config/settings.yml

```diff
@@ -1238,6 +1238,7 @@
       :performance_rollup_purging_start_delay: 5.minutes
       :policy_events_purge_interval: 1.day
       :poll: 15.seconds
+      :queue_timeout_interval: 1.minute
```

Comment:
Still not sure 1 minute is enough, but we can continue to debate even after merge.

Reply:
What if we did 15 seconds?  That's *basically* what it was doing before, right?